src/server/ApiRoutes/Bib.js

```diff
@@ -4,11 +4,13 @@ import nyplApiClient from '../routes/nyplApiClient';
 import logger from '../../../logger';
 import appConfig from '../../app/data/appConfig';
 import extractFeatures from '../../app/utils/extractFeatures';
+import { itemBatchSize } from '../../app/data/constants';
 
-const nyplApiClientCall = (query, urlEnabledFeatures) => {
+const nyplApiClientCall = (query, urlEnabledFeatures, itemFrom) => {
   // If on-site-edd feature enabled in front-end, enable it in discovery-api:
+  const queryItemPage = typeof itemFrom !== 'undefined' ? `?items_size=${itemBatchSize}&items_from=${itemFrom}` : '';
```

Comment:
`itemPageQuery` or `queryForItemPage`?

Reply:
I don't have strong feelings about these names. I've changed it to `queryForItemPage` since that seems a bit more self-describing, but happy to go either way.